app/components/settings/area-detail/alert-system/dataset-options/index.js

```diff
@@ -39,17 +38,11 @@ class DatasetOptions extends Component {
           </View>
         </View>
         <View style={[styles.row, styles.nested]}>
-          {(slug !== 'viirs') ?
-            <TimeFramePicker
-              startDate={startDate}
-              endDate={endDate}
-              updateDate={this.handleUpdateDate}
-            /> :
-            <DropdownPicker
-              updateViirsRange={this.handleUpdateDate}
-              endDate={endDate}
-            />
-          }
+          <DropdownPicker
+            updateRange={this.handleUpdateDate}
+            startDate={parseInt(startDate, 10)}
```

Comment:
It is ok for now but we have to think a way to unify viirs and glad alerts dates system also better names to make this easily understandable.